lib/pbench/server/database/alembic/versions/8e43d4c470a7_rename_api_key.py

```diff
@@ -0,0 +1,38 @@
+"""Rename column api_key to key
+
+Revision ID: 8e43d4c470a7
+Revises: 1a91bc68d6de
+Create Date: 2023-05-03 18:55:50.094026
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "8e43d4c470a7"
+down_revision = "1a91bc68d6de"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Did you adjust them?  If not...should you?  If so (and you think they are correct), please remove this comment.

Ditto for the one at line 28.

Reply:
Including *two* new alembic version files in a single PR is ... odd ... and ought to be completely unnecessary. I thought you were going to edit the previous one? 😕